libnopegl/src/nopegl.h.in

```diff
@@ -118,11 +132,19 @@ struct ngl_scene {
 NGL_API struct ngl_scene *ngl_scene_create(void);
 
 /**
- * Initialize the scene graph passing down the graph directly.
+ * Initialize the scene from parameters.
+ *
+ * The reference counter of the root node will be incremented for the scene
+ * lifetime, or until another scene replaces it.
+ */
+NGL_API int ngl_scene_init(struct ngl_scene *s, const struct ngl_scene_params *params);
+
+/**
+ * Return a pointer to the parameters of the scene.
```

Comment:
Ditto.

Reply:
This is actually weird because there is no description anymore but a note about the return that comes after:
```c
/**
 * The lifetime of the pointer matches the lifetime of the scene object.
 *
 * @return a pointer to the parameters of the scene
 */
NGL_API const struct ngl_scene_params *ngl_scene_get_params(const struct ngl_scene *s);
```

How would you deal with that?

Reply:
/**
 * Get the parameters of a scene.
 * 
 * The lifetime of the pointer matches the lifetime of the scene object.
 *
 * @return a pointer to the scene parameters
 */

Reply:
Thanks, changed. I ignored the `s` param though, it's pretty obvious in this context (it's not done for all the other functions)

Reply:
Can you add it anyway ? The other functions can be updated in a later PR.

Reply:
done, see also other consistency commits